.travis.yml

```diff
@@ -1,38 +1,47 @@
 ---
-language: node_js
```

Comment:
Restore the global jobs pls. It does work. You need to fix this.

Reply:
Ok

Reply:
You have hint on how can i do it?

Reply:
#40 is now merged.  Please rebase